src/Microsoft.AspNet.Mvc.ModelBinding/Metadata/DefaultModelMetadata.cs

```diff
@@ -48,7 +48,7 @@ public IReadOnlyList<object> Attributes
         {
             get
             {
-                return _details.Attributes;
+                return _details.Attributes.Attributes;
```

Comment:
`ModelAttributes` now available throughout the metadata details processing but not after.  should the whole thing also be available in `ModelMetadata`?